DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs

```diff
@@ -12,6 +12,11 @@ public class CourseContent
         public string? BannerText { get; }
         public bool IncludeCertification { get; }
         public DateTime? Completed { get; }
+        public int AssessAttempts { get; }

```

Comment:
This is specifically post learning assessment attempts right? If so, can we make that clear in the name? 🙂 

Oh and it's also actually a max assessment attemps right? So worth making that clear in the name as well 🙂

Reply:
I think so, it's a field in `Applications`. Would something like `MaxPostLearningAssessmentAttempts` be good? 🙂

Reply:
Yes, that sounds good to me 🙂

Reply:
Done in 86103de 🙂